authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -91,9 +94,42 @@ protected RangerAuthorizationPlugin(String metalake, Map<String, String> config)
     rangerAdminName = config.get(RangerAuthorizationProperties.RANGER_USERNAME);
     // Apache Ranger Password should be minimum 8 characters with min one alphabet and one numeric.
     String password = config.get(RangerAuthorizationProperties.RANGER_PASSWORD);
-    rangerServiceName = config.get(RangerAuthorizationProperties.RANGER_SERVICE_NAME);
+
+    String serviceName = config.get(RangerAuthorizationProperties.RANGER_SERVICE_NAME);
     rangerClient = new RangerClientExtension(rangerUrl, authType, rangerAdminName, password);
 
+    if (Boolean.parseBoolean(
+        config.get(RangerAuthorizationProperties.RANGER_SERVICE_CREATE_IF_ABSENT))) {
```

Comment:
How can we be so sure that this property is there?

Reply:
Boolean.parseBoolean can handle null value. Null value equals false.

Reply:
Please make sure there won't be uncaught exceptions.

Reply:
You can see the code
```
    public static boolean parseBoolean(String s) {
        return ((s != null) && s.equalsIgnoreCase("true"));
    }
```